process/elasticproc/logsevents/logsAndEventsProcessor.go

```diff
@@ -296,6 +282,20 @@ func (lep *logsAndEventsProcessor) getOriginalTxHash(logHashHex string) string {
 	return ""
 }
 
+func (lep *logsAndEventsProcessor) getExecutionOrder(logHashHex string) int {
+	tx, ok := lep.logsData.txsMap[logHashHex]
+	if ok {
+		return tx.ExecutionOrder
+	}
+
+	scr, ok := lep.logsData.scrsMap[logHashHex]
+	if ok {
+		return scr.ExecutionOrder
+	}
+
+	return 0
```

Comment:
maybe -1 to signal an error computing the execution order?

Reply:
changed to -1 and added an extra WARN